test/functional/page_objects/discover_page.js

```diff
@@ -61,8 +71,11 @@ export function DiscoverPageProvider({ getService, getPageObjects }) {
     }
 
     async openSavedSearch() {
-      await this.clickLoadSavedSearchButton();
-      await testSubjects.exists('loadSearchForm');
+      await retry.try(async () => {
```

Comment:
Can we leave a comment here about how this is flaky without the `retry.try`?

Reply:
yes, great idea!  I'll get that in.